### ISSUE TITLE
Add Camper profile

### ISSUE THREAD

**Issue by orson-69** [GIScience/openrouteservice#1063]
First of all I apologize because I don't know English, I use Google Translate.
My request is to insert the camper profile in the profiles in https://maps.openrouteservice.org/.
It would also be good to have predefined profiles as dimensions, so it might be better to insert two profiles.
One with Motorhomes and one for Campervans
For example, the Motorhome could have default dimensions in meters.
Height: 2.70 Length: 6.50 Width 2.05
Campervan:
Height: 3.20 Length: 7.20 Width: 2.3

**Comment by koebi** [comment]
Given that there's a [specific tag](https://wiki.openstreetmap.org/wiki/Key:motorhome) for this, it might indeed be beneficial to have a _private_ heavy vehicle profile to set length, width and the likes.

**Comment by koebi** [comment]
There are only about ~2000 uses of the `motorhome` tag on the openstreetmap, so I don't expect that this is something to be supported soon.

However, adding dimensional restrictions to the car profile is definitely something that we're considering.
Unfortunately, this will require some restructuring of how the ors stores information about roads, so it will take some time until this happens.

**Comment by orson-69** [comment]
Yes.
However, that is a tag used only to signal the possibility of access to services by Motorhomes.
There are other Keys that are used to indicate specific services, and these are more common
The parking areas are about 19,000
https://wiki.openstreetmap.org/wiki/Tag:tourism%3Dcaravan_site
There are 5,400 loading / unloading stations
https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dsanitary_dump_station
These indicators give more the idea of ​​the importance of these means, as diffusion.
At least in Europe they are growing rapidly in terms of diffusion


However, I would not take care of it starting from the cars, and putting the dimensions to the same.
Motorhomes are more similar to delivery truck profiles for which profiles already exist, and for these vehicles the possibility of entering dimensions and weight already exists.

Basically the simplest solution would be to create another profile by taking the data of the Delivery Trucks and proposing, if possible already default dimensions typical of these vehicles.

Maybe if you create a category it could be divided into 3 subcategories.
Van (common mainly in the USA)
Motorhomes (Class A classified in the USA)
Motorhomes (in the USA classified Class C)

**Comment by koebi** [comment]
Hey,

thanks for you comments :) We know of the importance and growth of motorhome infrastructure.
However, both the `tourism:caravan_site` and the `amenity:sanitary_dump_station`-tags are not used by the openrouteservice.

They are important for the user, but have no use when calculating routes. For this, only the `access:*`-tags such as `motorhome` are taken into account.

Since motorhomes are most similar to `hgv`s, one of these profiles could be used to route. The main difference between one of the `hgv`-profiles and the `car`-profile is the fact that access tags such as `hgv:no` and vehicle dimensions are taken into account.

A motorhome profile would do the same, but not respect `hgv:no` but instead have to respect `motorhome:no` access tags, which is currently not used by the openrouteservice.
As stated, there are too few instances of this tag to justify its inclusion (especially since roughly half of the tags are `motorhome:yes`).

We think that users would mostly benefit by being able to state their vehicle dimensions and find routes suitable to their vehicle.
As of now, this could probably be done by a creative use of the `driving-hgv`-parameter `"options":{"vehicle_type":"goods"}`, which will only respect the `goods:no`-tag, which is way less prevalent than the `hgv:no`-tag.
Users can then enter the specific dimensions of their vehicle. We would not add any default values, since they would probably not apply to the majority of users but pretend to be _reasonable_, and might lead to grave problems with the resulting routes.

Since stating dimensions does not only help motorhome-drivers, but is also helpful for anyone driving a longer/wider-than-usual car or maybe carrying building materials or any sort of trailer, this is an addition to the car-profile that we're thinking about but that will be delayed for the reasons above.

Best regards

**Comment by aoles** [comment]
Closing it for now, as there are no plans to implement an RV-specific profile in the foreseeable feature. However, some of the upcoming features might facilitate routing with motorhomes, campers and trailers, in particular https://github.com/GIScience/openrouteservice/issues/1314.
